src/bundle/Controller/REST/ContentController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClientBundle\Controller\REST;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use EzSystems\EzPlatformRest\Server\Controller as RestController;
+use EzSystems\EzPlatformRest\Server\Exceptions\AuthenticationFailedException;
+use EzSystems\EzRecommendationClient\Authentication\AuthenticatorInterface;
+use EzSystems\EzRecommendationClient\Service\ContentServiceInterface;
+use EzSystems\EzRecommendationClient\Value\Content;
+use EzSystems\EzRecommendationClient\Value\ContentData;
+use Ibexa\Personalization\QueryType\ContentQueryType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ContentController extends RestController
+{
+    /** @var \eZ\Publish\API\Repository\Repository */
+    protected $repository;
```

Comment:
Why `protected`instead of `private` ?

Reply:
This is required by `EzSystems\EzPlatformRest\Server\Controller` If we change it to private then we got:  "Access level to 'ContentController::$repository' must be 'protected' (as in class 'Controller') or weaker"

Reply:
If so, why do we even declare this property though? :thinking:

Reply:
It seems is no explanation why this property is there. I will remove it.

Reply:
Fixed: https://github.com/ezsystems/ezrecommendation-client/pull/106/commits/104a30c37169ef64d42fc4fa1229e13a65432a81